OTRExporter/OTRExporter/ArrayExporter.cpp

```diff
@@ -32,7 +32,7 @@ void OTRExporter_Array::Save(ZResource* res, const fs::path& outPath, BinaryWrit
 			writer->Write((uint32_t)vec->scalarType);
 			writer->Write((uint32_t)vec->dimensions);
 
-			for (int k = 0; k < vec->dimensions; k++)
+			for (uint32_t k = 0; k < vec->dimensions; k++)

```

Comment:
Why not `size_t` here?

Reply:
I think this would cause a truncation warning but I'm not 100% sure, it'd have to check.

Reply:
Actually `size_t` works fine here. Not sure why I made this change.